dask_ml/model_selection/_incremental.py

```diff
@@ -844,6 +844,37 @@ def _get_params(self):
             )
 
     def _additional_calls(self, info):
+        calls = {k: v[-1]["partial_fit_calls"] for k, v in info.items()}
+
+        patience_calls = max(int(self.patience) // 3, 1)
```

Comment:
I chose 3 because I think that's sufficient to detect a plateau. I think 4 scores within `tol` and `patience` is sufficient to say a plateau has been reached.

Lower values mean plateau detection is more accurate. Higher values mean shorter time to solution (`best_estimator_`).

Reply:
The docs for patience say

```
    patience : int, default False
        Maximum number of non-improving scores before we stop training a
        model. Off by default.
```

I guess I'm not sure why patience_calls isn't just `patience`.

Reply:
> I guess I'm not sure why patience_calls isn't just patience.

Good point. I chose `patience // 3` to protect against the case when scoring is noisy. I think with the documentation below, `patience_calls` should be `patience`.

    patience : int, default False
        If specified, training stops when the score does not increase by
        ``tol`` after ``patience`` calls to ``partial_fit``. Off by default.

Reply:
Ah, right. [Scikit-Learn's SGDClassifier][2] and [PyTorch's ReduceLROnPlateau][2] score after every `partial_fit` call, and stop training only if *all* of the most recent `patience` scores haven't improved by `tol`.

I implemented `patience_calls = int(self.patience) // 3` to balance between sampling after *every* `partial_fit` call and only after the *last* one. I did this because there is a larger cost to evaluating the validation set with Dask's parallel architecture.

`patience_calls = int(self.patience)` has clear documentation and provides the same benefit, though it might require a slightly higher `tol`. It provides clear performance benefits with Dask's parallel architecture.

[1]:https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau
[2]:https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html